include/solvers/mpm_base.tcc

```diff
@@ -231,6 +231,9 @@ void mpm::MPMBase<Tdim>::initialise_mesh() {
   // Read nodal euler angles and assign rotation matrices
   this->node_euler_angles(mesh_props, mesh_io);
 
+  // Read and assign acceleration constraints
+  this->nodal_acceleration_constraints(mesh_props, mesh_io);
+
   // Read and assign velocity constraints
   this->nodal_velocity_constraints(mesh_props, mesh_io);
```

Comment:
Can we assign the acceleration constraints after velocity constraints, i.e. move L235 after L238? If that's ok, let's move the check inside the `assign_acceleration_constraint` instead of `assign_velocity_constraint`. That way, we don't need to check every time we call `assign_velocity_constraint`, which is used and called more often than `assign_acceleration_constraint`.

Reply:
moved